akka-http-core/src/main/java/akka/http/javadsl/model/headers/ContentLength.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.http.javadsl.model.headers;
+
+/**
+ *  Model for the `Content-Length` header.
+ *  Specification: https://tools.ietf.org/html/draft-ietf-httpbis-p1-messaging-26#section-3.3.2
+ */
+public abstract class ContentLength extends akka.http.scaladsl.model.HttpHeader {
```

Comment:
All headers in `akka.http.javadsl.model.headers` seem to extend `akka.http.scaladsl.model.HttpHeader`.